package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@mapbox/frontend-documentation",
+  "version": "0.0.1",
+  "description": "Mapbox frontend tools for documentation websites.",
+  "main": "index.js",
+  "scripts": {
+    "start": "budo ./src/test-cases-app/test-cases-app.js --dir ./src/test-cases-app --live --debug --pushstate -- -t babelify",
+    "cp-assembly": "del ./src/test-cases-app/dist && cpy node_modules/@mapbox/mapbox-assembly/dist/*.* ./src/test-cases-app/dist",
+    "budo": "budo test/app.js -l -d -P -- -t babelify",
+    "prestart": "npm run build-component-index && npm run cp-assembly",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build-component-index": "rtk-index ./src ./src/test-cases-app/component-index.js",
+    "lint": "eslint .",
+    "pretest": "npm run lint",
+    "precommit": "lint-staged",
+    "format": "prettier '**/*.js' --write",
+    "build": "babel src/components -d components"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mapbox/frontend-documentation.git"
+  },
+  "author": "Mapbox",
+  "license": "BSD-2-Clause",
+  "bugs": {
+    "url": "https://github.com/mapbox/frontend-documentation/issues"
+  },
+  "homepage": "https://github.com/mapbox/frontend-documentation#readme",
+  "devDependencies": {
+    "@mapbox/eslint-config-mapbox": "^1.2.1",
+    "@mapbox/mapbox-assembly": "^0.26.0",
+    "@mapbox/react-test-kitchen": "^0.1.2",
+    "babel-cli": "^6.26.0",
+    "babel-eslint": "^8.2.5",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.24.1",
+    "babelify": "^8.0.0",
+    "budo": "^11.3.2",
+    "cpy": "^7.0.1",
+    "cpy-cli": "^2.0.0",
+    "del": "^3.0.0",
+    "del-cli": "^1.1.0",
+    "eslint": "^5.1.0",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-import": "^2.13.0",
+    "eslint-plugin-node": "^6.0.1",
+    "eslint-plugin-promise": "^3.8.0",
+    "eslint-plugin-react": "^7.10.0",
+    "globby": "^8.0.1",
+    "husky": "^0.14.3",
+    "lint-staged": "^7.2.0",
+    "lodash": "^4.17.10",
+    "nanohref": "^3.0.3",
+    "nanorouter": "^3.1.1",
+    "pascal-case": "^2.0.1",
+    "pify": "^3.0.0",
+    "prettier": "^1.13.7",
+    "prop-types": "^15.6.2",
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1",
+    "react-test-renderer": "^16.4.1"
```

Comment:
Looks like there a number of `devDependencies` that can be removed. The mapbox-react-components one these are from needs a little pruning, too, but it has tests and docs that this repo doesn't have, so not all its dependencies are relevant.